src/app/pages/auth/components/welcome-page/welcome-page.component.html

```diff
@@ -19,13 +24,17 @@
 
       <ion-scroll
         scrollY="true"
-        class="scroll-page"
+        class="scroll-page" style="height: calc(100% - 16px) !important;"
```

Comment:
why not add the height to the css file in the scroll-page class?

Reply:
I didn't check the structure of css files. Just to make it dominant, put inline-style. I can relocate all inline-styles in css files but needs to test the final view again.